e2e/src/utils/search.utils.ts

```diff
@@ -156,3 +152,11 @@ const buildNotSelector = (notSelectors: OtherSelectors) => {
   const pairs = buildSelectorPairs(notSelectors);
   return `:not([${pairs.join(']):not([')}])`;
 };
+
+export const clearDataFromInput = async () => {
+  await BrowserContext.page.keyboard.press('End');
```

Comment:
Does "Ctrl+A", "Backspace" work in the tests? If yes, consider replacing this keypresses combo; otherwise, just resolve the conversation